.readthedocs.yml

```diff
@@ -0,0 +1,9 @@
+---
+version: 2
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.11"
+  commands:
+    - pip install -U nox
+    - 'nox -e mkdocs -- build --site-dir _readthedocs/html "${READTHEDOCS_OUTPUT}/html"'
```

Comment:
```suggestion
    - mkdir -p _readthedocs/html/
    - nox -e mkdocs -- build --site-dir=_readthedocs/html
```

Reply:
Why is this preferable to the current approach?

Reply:
Hi. I haven't tried building this on readthedocs and I could be off. That said I believe it's necessary to create the directory (see [RTD docs](https://docs.readthedocs.io/en/stable/reference/environment-variables.html#envvar-READTHEDOCS_OUTPUT) about this).

Secondly I'm not sure why you need `_readthedocs/html "${READTHEDOCS_OUTPUT}/html"`